app/bundles/CoreBundle/Assets/js/4.builder.js

```diff
@@ -1,184 +1,5 @@
-                }
-                // Clear the customize forms
-                mQuery('#slot-form-container, #section-form-container').html('');
-                Mautic.inBuilderSubmissionOn(form);
-                var bgApplyBtn = mQuery('.btn-apply');
-                if (0 === bgApplyBtn.length && ("1" === Mautic.getUrlParameter('contentOnly') || Mautic.isInBuilder)) {
-                    var frm = mQuery('.btn-save').closest('form');
-                    Mautic.inBuilderSubmissionOn(frm);
-                    frm.submit();
-                    Mautic.inBuilderSubmissionOff();
-                } else {
-                    bgApplyBtn.trigger('click');
-                }
-                Mautic.inBuilderSubmissionOff();
-            }, true);
-        } catch (error) {
-            Mautic.removeButtonLoadingIndicator(applyBtn);
-            if (/SYNTAX ERROR/.test(error.message.toUpperCase())) {
-                var errorMessage = 'Syntax error. Please check your HTML code.';
-                alert(errorMessage);
-                console.error(errorMessage);
-            }
-            console.error(error.message);
-        }
-    });
-
-    // Blur and focus the focussed inputs to fix the browser autocomplete bug on scroll
-    builderPanel.on('scroll', function(e) {
-        // If Froala popup window open
-        if(mQuery.find('.fr-popup:visible').length){
-            if(!Mautic.isInViewport(builderPanel.find('.fr-view:visible'))) {
-                builderPanel.find('.fr-view:visible').blur();
-                builderPanel.find('input:focus').blur();
-            }
-        }else{
-            builderPanel.find('input:focus').blur();
-
-        }
-    });
-
-    var overlay = mQuery('<div id="builder-overlay" class="modal-backdrop fade in"><div style="position: absolute; top:' + spinnerTop + 'px; left:' + spinnerLeft + 'px" class="builder-spinner"><i class="ri-loader-3-line ri-spin ri-5x"></i></div></div>').css(builderCss).appendTo('.builder-content');
-
-    // Disable the close button until everything is loaded
-    btnCloseBuilder.prop('disabled', true);
-    applyBtn.prop('disabled', true);
-
-    // Insert the Mautic assets to the header
-    var assets = Mautic.htmlspecialchars_decode(mQuery('[data-builder-assets]').html());
-    themeHtml = themeHtml.replace('</head>', assets+'</head>');
-
-    Mautic.initBuilderIframe(themeHtml, btnCloseBuilder, applyBtn);
-};
-
-Mautic.isInViewport = function(el) {
-    var elementTop = mQuery(el).offset().top;
-    var elementBottom = elementTop + mQuery(el).outerHeight();
-
-    var viewportTop = mQuery(window).scrollTop();
-    var viewportBottom = viewportTop + mQuery(window).height();
-
-    return elementBottom > viewportTop && elementTop < viewportBottom;
+Mautic.launchBuilder = function () {
+    alert('Please enable the GrapesJS builder plugin to use this feature. Or other builder plugin.');
```

Comment:
```suggestion
    alert('Please enable the GrapesJS builder plugin (or another builder plugin) to use this feature.');
```
This needs to be rewritten. I'm suggesting the above change - what do you think? @RCheesley

Reply:
I think that's great, thanks @matbcvo